tests/EdFi.TestFixture/TestFixtureBase.cs

```diff
@@ -35,6 +35,7 @@ public virtual void RunOnceBeforeAny()
             try
             {
                 //Arrange
+                EstablishContext();
```

Comment:
The intent was not to modify this file, so what means any tests that had EstablishContext should be moved to Arrange, and ExecuteBehaviour should be moved to Act.